src/inputHandler.ts

```diff
@@ -38,6 +40,9 @@ export class InputHandler {
         window.addEventListener("keydown", (event) => {
             this.setArrow(event.key, true);
         });
+        this.hunterSpeedSlider.addEventListener("input", () => {
+            this.updateHunterSpeed(+this.hunterSpeedSlider.value, +this.hunterSpeedSlider.min, +this.hunterSpeedSlider.max);

```

Comment:
Can we avoid making implicit casts with `+foo`? That's very much a JavaScript idiom, but for TypeScript we want to be stricter with our type handling. What type does slider.value return? If it's a number we don't need to do anything, however if it's a string, I'd rather we have an explicit call to parseInt.